### DISCUSSION TITLE
Is it possible to show (column top) bar in gallery view?

### DISCUSSION THREAD

**Discussion by silvernull** [Taiko2k/Tauon#1100] [Q&A]
I'd like to be able to disable or enable certain fields while in gallery view,
specifically only view track title, length and if it's rated loved or not.

But I can't seem to get it to work.

**Comment by Taiko2k** [comment]
Column view will automatically deactivate if there is little horizontal space. What you need to do is resize the window wider or shrink the gallery panel, then reactivate column view.

**Comment by silvernull** [comment]
I apologise if I wasn't particularly clear, there is no documented name I can find for this thing, only references to it being called "column top bar"

but the top column bar, with the headings and info you can enable and disable (that doesn't seem to save when the application is reopened)
in the bottom view mode in MENU.
is completely unavailable in gallery view, even with plenty of space, that i would reduce to below the default gallery column space _anyway_

![image](https://github.com/Taiko2k/TauonMusicBox/assets/80038133/3ec027a1-f582-40d8-a699-c33284140133)

resizing my window then reducing it in size let me enabled both but that's a workaround

![image](https://github.com/Taiko2k/TauonMusicBox/assets/80038133/4ffccd47-55fb-41f3-9b72-6c3cbf4f2148)

**Comment by Taiko2k** [comment]
The “show/hide” top column bar will only show when column mode is active. I
think what I need to do is lower or remove entirely the automatic
deactivation threshold.

**Reply by silvernull** [reply]
I thought that might be the case from your previous reply.

**Reply by silvernull** [reply]
For some reason it's been working flawlessly now? For a few days.
With no update.
I do not know how or why, or what I did.
![image](https://github.com/Taiko2k/TauonMusicBox/assets/80038133/96c6c5a8-9f92-4909-8a34-e3b9689e7d9d)

**Comment by silvernull** [comment]
Also relating to the column bar.

The column tabs to the left and right of the one you are resizing should maintain their width (atleast to me instinctively,
since you are not resizing the entire column but expect to control the size of the individual tabs, like it does in file explorer)
![image](https://github.com/Taiko2k/TauonMusicBox/assets/80038133/d8c9ca48-ebd7-40f1-8459-4d69cf9711d4)

Otherwise it can make tabs go off the column entirely because it doesn't stretch to accommodate them.

Could easily be a setting

![image](https://github.com/Taiko2k/TauonMusicBox/assets/80038133/11905d4f-0016-492c-9b5e-780e6bc0410f)

**Comment by silvernull** [comment]
Thank you for responding, and so quickly.
